packages/atdgen-windows.1.10.0/files/atdgen.install

```diff
@@ -0,0 +1,6 @@
+bin: [
+  "src/atdgen.run" {"atdgen.run"}
+  "src/atdgen" {"atdgen"}
+  "atdgen-cppo/atdgen-cppo"
+  "atdgen-cppo/cppo-json"
+]
```

Comment:
I fully agree and I was about to remove the `atdgen.install` file when I realized that you kept `/packages/yojson-windows.1.3.2/files/yojson.install`. 
I was then confused but I leave there both `atdgen.install` and `atd.install`!